Task_Position_the_Cubes/index.css

```diff
@@ -1,27 +1,65 @@
-
-body {
-  background-color: lightblue;
-}
-p{
```

Comment:
The value 'center' is not a valid margin value. Please use 'auto' for horizontal margin and 'auto' for vertical margin.